app/components/KakaoLoginPage.tsx

```diff
@@ -0,0 +1,57 @@
+"use client";
+
+import React, { useCallback, useEffect, useState } from "react";
+import { Auth } from "@supabase/auth-ui-react";
+import { ThemeSupa } from "@supabase/auth-ui-shared";
+import useHydrate from "@/hooks/useHydrate";
+import { User } from "@supabase/supabase-js";
+import { createSupabaseBrowserClient } from "@/lib/supabase/browserClient";
+
+const KakaoLoginPage = () => {
+  const [user, setUser] = useState<User | null>(null);
+  const isMount = useHydrate();
+  // createSupabaseServerSideClient 왜 안되나
+  const supabase = createSupabaseBrowserClient();
+
+  const getUserInfo = useCallback(async () => {
+    const result = await supabase.auth.getUser();
+    if (result?.data?.user) setUser(result?.data?.user);
+  }, [supabase]);
+
+  const handleLogout = async () => {
+    await supabase.auth.signOut();
+    window.location.reload();
+    console.log(user);
+  };
+
+  useEffect(() => {
+    getUserInfo();
+  }, []);
```

Comment:
여기 타입 에러 안나나요?

```suggestion
  useEffect(() => {
    getUserInfo();
  }, [gaetUserInfo]);
```

원래 코드는 이렇게 했던거 같아서